### ISSUE TITLE
Error in inmuneConvoy.sqf

### ISSUE THREAD

**Issue by jahangir13** [A3Antistasi/A3-Antistasi#214]
*Version:* 1.4.0.2

*Mods:* CBA, RHS, ACE Antistasi Edition

*Environment*: MP dedi.

*.rpt attatched?* YES (screenshot)

*have you edited the missionfile?*: No

***Issue:***

I saw the following error in the Server log today. This happened a few minutes after everyone leaving the Server yesterday night. Not sure if it causes any bigger issue. 

![inimmuneconvoy_error](https://user-images.githubusercontent.com/7389834/49605460-d3463280-f990-11e8-81f8-fb65fab99375.jpg)

However, searching for the issue I found that the dirTo function (in line 62) should not be used anymore in A3:
https://community.bistudio.com/wiki/BIS_fnc_dirTo             (Description)

**Comment by Barbolani77** [comment]
Yes, that function should be replaced, but no worries abut an error there, it's harmless
